packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1054,6 +1062,18 @@ export class MergeTree {
 			const firstMoveSeqIdx = this.moveSeqs.findIndex((seq) => seq >= minSeq);
 			this.moveSeqs = firstMoveSeqIdx === -1 ? [] : this.moveSeqs.slice(firstMoveSeqIdx);
 			if (MergeTree.options.zamboniSegments) {
+				walkAllChildSegments(this.root, (seg) => {
+					const refs = seg.localRefs ?? [];
+					for (const obliterateRef of refs) {
```

Comment:
i call this out a bit above, but we need to limit this to just obliterates below the updated minseq, the current code removes all obliterates.